lib/storage/src/rbac/mod.rs

```diff
@@ -230,6 +245,8 @@ pub struct AccessRequirements {
     pub manage: bool,
     /// If true, the access should be not limited by a payload restrictions.
     pub whole: bool,
+    /// Require access to collection extras, like snapshots, payload indexes, cluster info.
+    pub extras: bool,
```

Comment:
This `extra` field feels a bit like a shortcut to cram things together.
Would it make sense to have dedicated flags for `snapshots`, `payload_index` and `cluster_info`?

Reply:
I though about it, but if I we do it like this, the next question would be - why not do it for everything?
This about it from the other side: we don't want to block a list of things, we want to allow access to points only. If we would have some new entity inside the collection, we won't need to come up with a new name, we just put it under extras.

Anyway, this is an internal logic, so I propose to keep it like this until we would need to change it.